packages/react-app/src/components/modals/schedule-claim-modal.tsx

```diff
@@ -210,11 +225,13 @@ export default function ScheduleClaimModal({
       <Formik
         initialValues={
           {
-            evidence: '',
-            claimedAmount: { parsedAmount: 0, token: questTotalBounty?.token } as TokenAmountModel,
-            claimAll: false,
-            contactInformation: undefined,
-            playerAddress: undefined,
+            evidence: claimDataState.evidence ?? '',
+            claimedAmount:
+              claimDataState.claimedAmount ??
+              ({ parsedAmount: 0, token: questTotalBounty?.token } as TokenAmountModel),
+            claimAll: claimDataState.claimAll ?? false,
+            contactInformation: claimDataState.contactInformation ?? undefined,
```

Comment:
Redundant because contactInformation can only be undefined or string
```suggestion
            contactInformation: claimDataState.contactInformation,
```

Reply:
oupsies ahaha